modin/pandas/test/test_series.py

```diff
@@ -2949,7 +2995,14 @@ def test_size(data):
 )
 def test_skew(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.skew(skipna=skipna), pandas_series.skew(skipna=skipna))
+    try:
+        pandas_result = pandas_series.skew(skipna=skipna)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.skew(skipna=skipna)
+    else:
+        modin_result = modin_series.skew(skipna=skipna)
+        df_equals(pandas_result, modin_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.skew(skipna=skipna))
```

The same comments as for `test_all` func

Reply:
Fixed.